src/app/shared/components/carousel/carousel2.0.html

```diff
@@ -1,46 +1,46 @@
-            <img  src="https://d1ih8jugeo2m5m.cloudfront.net/2021/06/Ecommerce-Thumbnail.jpg" style="width:100%" alt="">
-        </div>
-        <div class="mySlides">
-            <div class="numbertext">2/4</div>
-              <img  src="https://www.imagar.com/wp-content/uploads/2020/07/Desarrollo-web-ecommerce-1.jpg" style="width:100%" alt="">
-          </div>
-          <div class="mySlides">
-            <div class="numbertext">3/4</div>
-              <img  src="https://directivosygerentes.es/wp-content/uploads/2020/03/Ecommerce.jpg" style="width:100%" alt="">
-          </div>
-          <div class="mySlides">
-            <div class="numbertext">4/4</div>
-              <img  src="https://s3.amazonaws.com/cdn.wp.m4ecmx/wp-content/uploads/2019/04/25160328/El-crecimiento-del-eCommerce-en-M%C3%A9xico-compressor.jpg" style="width:100%" alt="">
-          </div>
-    </div>
-    <!-- Next and previous buttons -->
-    <a class="prev" (click)="plusSlides(-1)">&#10094;</a>
-    <a class="next" (click)="plusSlides(1)">&#10095;</a>
-  
-    <!-- Image text -->
-    <div class="caption-container">
-      <h2 id="caption"></h2>
-    </div>
-    <!-- Thumbnail images -->
-    <div class="row">
-      <div class="column">       
-        <img class="demo cursor" src="https://d1ih8jugeo2m5m.cloudfront.net/2021/06/Ecommerce-Thumbnail.jpg" style="width:100%" (click)="currentSlide(1)" alt="welcome">
+<div id="carouselExampleCaptions" class="carousel slide" data-bs-ride="false">
+  <div class="carousel-indicators">
+    <button type="button" (click)="showSlide(0)" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="carousel-indicator active" aria-current="true" aria-label="Slide 1"></button>
+    <button type="button" (click)="showSlide(1)"  data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" class="carousel-indicator" aria-label="Slide 2"></button>
+    <button type="button" (click)="showSlide(2)" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" class="carousel-indicator" aria-label="Slide 3"></button>
+    <button type="button" (click)="showSlide(3)" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3" class="carousel-indicator" aria-label="Slide 4"></button>
+  </div>
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img src="https://d1ih8jugeo2m5m.cloudfront.net/2021/06/Ecommerce-Thumbnail.jpg" class="d-block w-100" alt="...">
+      <div class="carousel-caption">
+        <h1>You can buy what you want</h1>
+        <h2>from the comfort of your own home</h2>
       </div>
-      <div class="column">       
-        <img class="demo cursor" src="https://www.imagar.com/wp-content/uploads/2020/07/Desarrollo-web-ecommerce-1.jpg" style="width:100%" (click)="currentSlide(2)" alt="b">
+    </div>
+    <div class="carousel-item">
+      <img src="https://www.imagar.com/wp-content/uploads/2020/07/Desarrollo-web-ecommerce-1.jpg" class="d-block w-100" alt="...">
```

Comment:
The carousel item should be its own component. The carousel height shouldn't change depending on the image, and the text should have a better appearance on the mobile version.